### DISCUSSION TITLE
神經網路介紹影片

### DISCUSSION THREAD

**Discussion by brian431** [1101-GS3073-C/Python-Community-GS3073-C#470] [General]
此一系列僅有四部影片，用稍微比較數學的角度講解神經網路是如何運作的。看完之後能對神經網路有基礎的了解，至少能知道裡面大概發生了什麼事情。想更進一步了解的話大概要去看一些開放式課程。

https://youtu.be/aircAruvnKk

**Comment by williamyen0623** [comment]
謝謝分享！了解神經網路的運作方式是建構良好機器學習模型的基礎！

**Comment by manipulateed** [comment]
神經網路算是比較進階的內容，能用數學角度去講解 很有幫助我們理解

**Comment by lx3211** [comment]
python很方便可以直接call函數庫，就不用理解背後的原理，但是如果以後想要活用自己做模型，就必須要知道背後的原理。

**Comment by Chenszuhong** [comment]
這有點難，但感謝分享

**Comment by limitx0** [comment]
3Blue1Brown 的解說真的淺顯易懂
